lms/envs/common.py

```diff
@@ -1939,3 +1939,7 @@
 
 #date format the api will be formatting the datetime values
 API_DATE_FORMAT = '%Y-%m-%d'
+
+
+############# OPEN EDX COURSE API #################
+FEATURES['COURSE_API'] = False
```

Comment:
Even though it's not needed, I don't think we should go out of our way to turn it off given that it's harmless. Lack of throttling shouldn't be a problem because of the header permission.